application/app/pom.xml

```diff
@@ -109,15 +109,38 @@
         <artifactId>jackson-annotations</artifactId>
         <version>2.12.3</version>
     </dependency>
+    <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-engine</artifactId>
+        <version>${junit-jupiter.version}</version>
+        <scope>test</scope>
+    </dependency>
+    <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-devtools</artifactId>
+        <optional>true</optional>
+    </dependency>
+    <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-junit-jupiter</artifactId>
+        <version>3.11.2</version>
+        <scope>test</scope>
+    </dependency>
 
```

Comment:
For test coverage wouldnt we want to use the plugin JaCoCo

Reply:
Sounds like an interesting thing to plug in. Lets do it.  Can we also please add a section in the readme.txt to say how to run it?

Reply:
I will add that in the upcoming stories